e2e-tests/portal/cypress/e2e/journeys/admin/create-user/create-a-user.spec.js

```diff
@@ -131,5 +141,50 @@ context('Admin user creates a new user', () => {
     });
   });
 
+  it('Admin user adds a new user using "{ "$gt": "" }", triggering validation error for email', () => {
+    // Login and go to the dashboard
+    cy.login(ADMIN);
+
+    header.users().click();
+    users.user(userWithInvalidPassword).should('not.exist');
+
+    users.addUser().click();
+
+    userWithInvalidPassword.roles.forEach((role) => {
+      createUser.role(role).click();
+    });
+
+    // as the string has object characters, need to use parseSpecialCharSequences
+    createUser.username().type(userWithInjection.username, { parseSpecialCharSequences: false });
+    createUser.manualPassword().click();
+    createUser.password().type(userWithInjection.password);
+    createUser.confirmPassword().type(userWithInjection.password);
+    createUser.firstname().type(userWithInjection.firstname);
+    createUser.surname().type(userWithInjection.surname);
+    createUser.bank().select(userWithInjection.bank);
+
+    createUser.createUser().click();
+
+    cy.url().should('eq', relative('/admin/users/create'));
+
+    // checks html form validation pop up contains correct error message
+    cy.get('input:invalid').should('have.length', 1);
+    createUser.username().then(($input) => {
+      expect($input[0].validationMessage).to.eq("Please include an '@' in the email address. '{\"$gt\":\"\"}' is missing an '@'.");
+    });
+
+    /**
+     * to check that user has not been created
+     * gets list of users from portal-api
+     * finds one with email { "$gt": "" }
+     * should be undefined
+     */
+    cy.listAllUsers().then((usersInDb) => {
+      const injectedUser = usersInDb.find((user) => user.email === userWithInjection.email);
```

Comment:
This needs to be updated to following as the user with email will be created with email as `email: {}`

```javascript
const injectedUser = usersInDb.find((user) => user.email === {});
```